docs/source/_templates/datasets.rst

```diff
@@ -0,0 +1,9 @@
+{% for item in data %}
+.. toctree::
+
+    :hidden:
```

Comment:
Hides the dataset entries for now. This means they are not listed in any table of contents. The pages are still accessible via explicit URLs.